src/styles/main.scss

```
@@ -1,3 +1,59 @@
-body {
+@use "sass:list";
+@import './variables';
+@import './reset';
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  box-sizing: border-box;
+  gap: $block-gap $block-gap;
+  width: 706px;
+  margin: 0 auto;
+
+  @each $length in $month-lengths {
+    &--month-length-#{$length} .calendar__day {
+      &:nth-child(n + #{$length + 1}) {
+        display: none;
+      }
+    }
+  }
+}
+
+.calendar__day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  width: $block-size;
+  height: $block-size;
+  box-sizing: border-box;
   margin: 0;
+  background-color: #eee;
+  border: 1px solid black;
+  transition-duration: 0.5s;
```

don't use it like that, because it's the same as transition: all 0.5s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transition-duration, transform $transition-duration;
```